flutter_fimber/lib/flutter_fimber.dart

```diff
@@ -155,9 +155,11 @@ class DebugBufferTree extends DebugTree {
         logLevels: logLevels, printTimeType: DebugTree.timeElapsedType);
   }
 
-  /// prints log line with `debugPrint`.
+  /// prints log lines breaking them into multiple lines if its too long.
+  /// src: https://github.com/flutter/flutter/issues/22665#issuecomment-458186456
   @override
   void printLog(String logLine, {String level}) {
-    debugPrint(logLine);
+    final pattern = RegExp('.{1,800}'); // 800 is the size of each chunk
```

Comment:
Add additional parameter and property into the constructor, and also as variable `var maxLineSize=-1;`
When:
-  `-1` means no new line - let it reach maximum.
- any other value - will chunk it to a value.

In case that value is more then max - log a ERROR level log entry.

Reply:
@magillus I made the changes, Im not sure if the comments are clear enough, my english may fail sometimes :P